listener/listener.go

```diff
@@ -320,8 +322,14 @@ func ensureSubscription(ctx context.Context, sm *servicebus.SubscriptionManager,
 	if err == nil {
```

Comment:
we are returning it once the subscription is already there, this is sad when we want to do some management operations. it'll be good if "sm.Put" can update the setting, we can let user choose when to update, Otherwise may need to find some way to clean the existing subscriptions.

Reply:
Agree though punting to a different change. For now would have to change subscription name,